p2p/peer_tracker.go

```diff
@@ -99,6 +99,16 @@ func (p *peerTracker) track() {
 	}
 }
 
+func (p *peerTracker) GetPeers() []peer.ID {
+	p.peerLk.RLock()
+	defer p.peerLk.RUnlock()
+	peers := make([]peer.ID, 0, len(p.trackedPeers))
+	for peerID := range p.trackedPeers {
+		peers = append(peers, peerID)
```

Comment:
same comment here, if we are pre-allocating, then we should make use of an index to fill the slice instead of continuing to re-allocate with `append`

Reply:
same